core/modules/SConscript

```diff
@@ -95,6 +95,14 @@ def standardModule(env, exclude=None, module=None, test_libs=None, unit_tests=No
                 then use this parameter to pass list (or space-separated
                 string) of application names. Can be empty string or list to
                 not run any unit tests.
+    @param bin_cc_targets: a dictionary defining whgich binary products (not
```

Comment:
whgich -> which
Not shure `targets` has a meaning here. The key in dict is a name of the source, not target?

Reply:
I'll come up with a better name